### PR TITLE
Enable access to non-public elements of java.lang

### PR DESCRIPTION
**Pull request by didiksupriadi41**
This should fix `xmvn-connector-ivy` for builds with JDK17.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,6 +83,9 @@
     <easymockVersion>4.3</easymockVersion>
     <compilerPluginVersion>3.8.1</compilerPluginVersion>
     <surefireVersion>3.0.0-M5</surefireVersion>
+    <argLine>
```
Comment (mizdebsk) [inline pom.xml:86]: argLine should not be a global property, but under configuration of maven-surefire-plugin.

**Review by mizdebsk — CHANGES_REQUESTED** [review]
The underlying issue is https://github.com/easymock/easymock/issues/274

I'm fine with adding a temporary workaround, but please mark it as such with a comment, so that it is removed once easymock is updated.

**Comment by didiksupriadi41** [comment]
I've moved `argLine` under configuration of maven-surefire-plugin and mark it with a comment.

**Comment by mizdebsk** [comment]
LGTM. Started automated testing.

**Review by mizdebsk — APPROVED** [review]

**Comment by mizdebsk** [comment]
Testing passed, merged.

**Pull request merged**